src/Controller/TideApiController.php

```diff
@@ -385,7 +385,8 @@ protected function initializeCacheId($path, Request $request) {
     // Dispatching GetCacheIdEvent event to give other modules an opportunity to
     // alter it.
     $event = $this->eventDispatcher->dispatch(TideApiEvents::GET_CACHE_ID, $event);
-    return $event->getCacheId();
```

Comment:
Is controlling logic looking at both `$this->cacheId` and the returded id? It should actually be one or the other. Maybe it shouldn't return anything?

Reply:
yes. @anthony-malkoun . thanks. i delete this return.